test/Microsoft.Sbom.Parsers.Spdx30SbomParser.Tests/Parser/GeneratorTests.cs

```diff
@@ -181,6 +181,69 @@ public void GenerateJsonDocumentTest_Relationship()
         Assert.AreEqual(expectedJsonContentAsString, generatedJsonString);
     }
 
+    [TestMethod]
+    public void GenerateJsonDocumentTest_PrereqFor_Relationship()
+    {
+        var relationshipInfo = new Relationship
+        {
+            SourceElementId = "source-id",
+            TargetElementId = "target-id",
+            RelationshipType = RelationshipType.PREREQUISITE_FOR,
+        };
+
+        var generatorResult = generator.GenerateJsonDocument(relationshipInfo);
+        var generatedJsonString = generatorResult.Document.RootElement.GetRawText();
+        generatedJsonString = NormalizeString(generatedJsonString);
+
+        var expectedJsonContentAsString = SbomRelationshipJsonStrings.RelationshipPrereqForJsonString;
+        expectedJsonContentAsString = NormalizeString(expectedJsonContentAsString);
+
+        Assert.IsFalse(generatedJsonString.Contains("null"));
+        Assert.AreEqual(expectedJsonContentAsString, generatedJsonString);
+    }
+
+    [TestMethod]
+    public void GenerateJsonDocumentTest_DescribedBy_Relationship()
+    {
+        var relationshipInfo = new Relationship
+        {
+            SourceElementId = "source-id",
```

Comment:
Rather than use "source-id" and "target-id" as specific strings, it might make sense to make them `const` values in the class. That's doubly true if you use the `[DataRow]` mentioned above